creator-node/src/services/sync/secondarySyncFromPrimaryUtils.ts

```diff
@@ -0,0 +1,112 @@
+import type {
+  ForceResyncAuthParams,
+  ForceResyncConfig,
+  ForceResyncSigningData,
+  SyncRequestAxiosData
+} from '../stateMachineManager/stateReconciliation/types'
+
+const _ = require('lodash')
+
+const { logger: genericLogger } = require('../../logging')
+const config = require('../../config')
+const ContentNodeInfoManager = require('../stateMachineManager/ContentNodeInfoManager')
+const {
+  recoverWallet,
+  generateTimestampAndSignature
+} = require('../../apiSigning')
+
+const generateDataForSignatureRecovery = (body: SyncRequestAxiosData) => {
+  const { wallet, creator_node_endpoint, sync_type, immediate } = body
+  return {
+    wallet,
+    creator_node_endpoint,
+    sync_type,
+    immediate
+  }
+}
+
+// Function to sign sync data. This is used to determine whether or not to `forceResync`
+const signSyncData = (syncData: ForceResyncSigningData) => {
+  return generateTimestampAndSignature(
+    syncData,
+    config.get('delegatePrivateKey')
+  )
+}
+
+// Derive the Content Node delegate wallet from the data, signature, and timestamp
+const recoverWalletFromSyncData = ({
+  data,
+  timestamp,
+  signature
+}: ForceResyncAuthParams) => {
+  return recoverWallet({ ...data, timestamp }, signature)
+}
```

Comment:
i dont think we need this function, its just a one liner calling `recoverWallet` which is correctly abstracted since the params are not actually "sync data" but signature data, same with `signSyncData` above, lets just use the `apiSigning` functions directly